native/src/modules/common/components/NewsHeader.js

```diff
@@ -24,8 +24,7 @@ const TouchableWrapper = styled.TouchableOpacity`
 const LocalTabWrapper: StyledComponent<
   { isSelected: boolean },
   ThemeType,
-  *
-> = styled.View`
+  *> = styled.View`
```

Comment:
why were there line breaks in the first place? I feel like you could pack the type in one line

Reply:
Different styling by the people from kiitos I guess.